client/js/app.js

```diff
@@ -0,0 +1,35 @@
+const btn = document.querySelector('#post-form');
+
+function submitPost(e){
+    e.preventDefault();
+    const postData = {
+        title: e.target.title.value,
+        name: e.target.name.value,
+        body: e.target.body.value
+    }
+
+    const options = {
+        method: 'POST',
+        body: JSON.stringify(postData),
+        redirect:  'follow',
+        headers: {
+            "Content-Type": "application/json"
+        }
+    };
+
+    fetch('http://localhost:3000/posts/', options)
+        //.then(d=>d.json())
+        .then(window.location.href = 'www.google.com')
```

Comment:
im curious why it gets redirected to google.com

Reply:
wrong version, mb